vnext/Microsoft.ReactNative/IReactNotificationService.idl

```diff
@@ -8,96 +8,87 @@ import "IReactPropertyBag.idl";
   {
-    // Subscribe to a notification.
-    // The notificationName as a property name can belong to a specific namespace. It must be not null.
-    // The dispatcher is used to call notification handlers. If it is null, then handler is called synchronously.
-    // The handler is a delegate that can be implemented as a lambda to handle notifications.
-    // The method returns IReactNotificationSubscription that must be kept alive while the subscription
-    // is active. The subscription is removed when the IReactNotificationSubscription is destroyed.
-    DOC_STRING("Subscribe to a notification. \
-The `notificationName` is a non-null property name and can belong to a specific namespace. \n\
-The dispatcher is used to call notification handlers. If it is null, then the handler is called synchronously. \
-The handler is a delegate that can be implemented as a lambda to handle notifications. \
-The method returns a @IReactNotificationSubscription that must be kept alive while the subscription \
-is active. The subscription is removed when the @IReactNotificationSubscription is destroyed.")
+    DOC_STRING(
+      "Subscribes to a notification.\n"
+      "The `notificationName` is a non-null property name and can belong to a specific namespace.\n"
+      "The `dispatcher` is used to call notification handlers. If it is null, then the handler is called synchronously.\n"
+      "The `handler` is a delegate that can be implemented as a lambda to handle notifications.\n"
+      "The method returns a @IReactNotificationSubscription that must be kept alive while the subscription "
+      "is active. The subscription is removed when the @IReactNotificationSubscription is destroyed.")
     IReactNotificationSubscription Subscribe(
       IReactPropertyName notificationName, IReactDispatcher dispatcher, ReactNotificationHandler handler);
 
-    // Send the notification with notificationName.
-    // The sender is the object that sends notification. It can be null.
-    // The data is the data associated with the notification. It can be null.
-    // Consider using IReactPropertyBag for sending semi-structured data. It can be created
-    // using the ReactPropertyBagHelper.CreatePropertyBag method.
-    DOC_STRING("Send the notification with `notificationName`. The sender is the object that sends notification and can be null. The `data` is the data \
-associated with the notification and can be null. Consider using @IReactPropertyBag for sending semi-structured data. \
-It can be created using the @ReactPropertyBagHelper.CreatePropertyBag method.")
+    DOC_STRING(
+      "Sends the notification with `notificationName`.\n"
+      "The `notificationName` must not be null.\n"
+      "The `sender` is the object that sends notification. It can be null.\n"
+      "The `data` is the data associated with the notification. It can be null.\n"
+      "Consider using @IReactPropertyBag for sending semi-structured data. It can be created "
```

Comment:
"`notificationName` must not be null.\n"
      "`sender` is the object that sends the notification. It can be null.\n"
      "`data` is the data associated with the notification. It can be null.\n"
      "Consider using @IReactPropertyBag for sending semi-structured data. It can be created "